src/iconsManifest/languages.ts

```diff
@@ -16,10 +16,7 @@ export const languages: ILanguageCollection = {
   autoit: { ids: 'autoit', defaultExtension: 'au3' },
   avro: { ids: 'avro', defaultExtension: 'avcs' },
   azcli: { ids: 'azcli', defaultExtension: 'azcli' },
-  azurepipelines: {
-    ids: 'azure-pipelines',
-    defaultExtension: 'azure-pipelines.yml',
-  },
+  azurepipelines: { ids: 'azure-pipelines', defaultExtension: 'azure-pipelines.yml'},
```

Comment:
Revert the formatting here.

Reply:
Please revert the formatting so we can proceed.
@robertohuertasm This is one case where the commit hook can't be in effect. The user edits the code from the browser editor.

Reply:
I just wanted to point out that we can't cover all cases.